src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeRelation.java

```diff
@@ -96,9 +98,16 @@ private RelationMemberList membersFor(final RelationBean bean)
         final List<RelationMember> memberList = new ArrayList<>();
         for (final RelationBeanItem item : bean)
         {
-            memberList.add(new RelationMember(item.getRole(),
-                    getChangeAtlas().entity(item.getIdentifier(), item.getType()),
-                    getIdentifier()));
+            final AtlasEntity memberChangeEntity = getChangeAtlas().entity(item.getIdentifier(),
+                    item.getType());
+            if (memberChangeEntity == null)
```

Comment:
Once we update the member fetching for `ChangeRelation`, seeing this exception will mean something went seriously wrong - ie. it's a good sanity check that our code is working correctly.